google/resource_container_cluster.go

```diff
@@ -742,3 +747,15 @@ func generateNodePoolName(prefix string, d *schema.ResourceData) (string, error)
 		return resource.UniqueId(), nil
 	}
 }
+
+func resourceContainerClusterStateImporter(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Test test is failling:
--- FAIL: TestAccGoogleContainerCluster_import (275.85s)
	testing.go:428: Step 1 error: Resource specified by ResourceName couldn't be found: google_container_cluster.np

The ID should be set to the name `only`.  See the create method above (line 474).

Reply:
Ahh.. Should be fixed now :)